tools/install.sh

```diff
@@ -115,37 +115,41 @@ echo "Installing JavaScript dependencies..." | print_info
 npm ci --no-fund
 echo "✔ Installed JavaScript dependencies" | print_success
 
-# Check if virtual environment exists
-if [[ -d ".venv" ]] && [[ "$(.venv/bin/python3 --version)" != "$(${PYTHON} --version)" ]]; then
-    echo "The given $(${PYTHON} --version) version differs from $(.venv/bin/python3 --version) of virtual environment." | print_warning
-    echo "Deleting the outdated virtual environment..." | print_info
-    rm -rf .venv
-fi
+if [ "$(whoami)" = "vscode" ]; then
+    echo "Skipping the initialization of the virtual environment because this devcontainer is already an isolated environment on its own." | print_info
+else
```

Comment:
There's at least one other place in the `require_installed` function  where the venv is being invoked. I'm not sure if it might not be easier to keep the venv-in-container setup, even if it's not really necessary.

(Unless there's a reason I don't know about here, of course)

Reply:
The reason was that the mentioned test support in vscode seemed to work without the virtual environment (venv) because it was able to find the correct test runner. However, it only executed a handful of tests and skipped the rest. 😢

Reply:
I reverted the commit so that we now use venv in devcontainer again. 👍

Reply:
Hm, weird. Does it not find the tests, or do they fail to setup? I've had a similar-ish issue with the flake, requiring to set two environment variables to properly find and run all tests:
```bash
export DJANGO_SETTINGS_MODULE="integreat_cms.core.settings"
export INTEGREAT_CMS_DEBUG=1
```

Reply:
The setup worked, as I suggested because some tests were successful, while others were found but marked as skipped. :/